client/src/pages/dashboard/Dashboard.js

```diff
@@ -77,13 +90,29 @@ function Dashboard() {
 
   return (
     <div className="dashboard">
-      <div className="navbar">this is the delicious navbar</div>
       <div className="dashboard-screen">
         <div className="proposal-list">
           <SearchBar keyword={input} setKeyword={updateInput}/>
           {proposalList}
         </div>
-        <div className="proposal-description"></div>
+        <div className="proposal-description">
+          <div className="proposal-background">
```

Comment:
For future improvements: add some dynamic width or something to this so that it scales well when you resize the screen